modules/configuring-your-ide.adoc

```diff
@@ -0,0 +1,199 @@
+:_module-type: PROCEDURE
+
+[id="configuring-your-ide_{context}"]
+= Configuring your IDE
+
+[role='_abstract']
+You can configure some notebook workbenches to get the most out of your data science work.
+
+== Configuring VS Code Server
```

Comment:
We should agree and unify the naming here. The app itself uses this: `VS Code for the Web`. Although we use `Code Server` as an image name. I think we should follow that the app says and use `Code Server` only in situation we refer to the name of our image?

Reply:
I think that's sensible. How should we handle references to 'your VS Code Server environment'? 

i.e. are those fine to leave as they are because it's the Code Server workbench environment, or should it be renamed to 'your VS Code for the Web environment'?